src/Apps/Sell/Components/SubmissionLayout.tsx

```diff
@@ -28,13 +28,13 @@ export const SubmissionLayout: React.FC<SubmissionLayoutProps> = ({
             </Column>
 
             <Column span={[8]}>
-              <Box maxWidth="100vw" width={CONTENT_WIDTH} p={2} pt={4}>
+              <Box maxWidth={CONTENT_WIDTH} p={2} pt={4}>
                 {children}
               </Box>
             </Column>
           </GridColumns>
         ) : (
-          <Box maxWidth="100vw" width={CONTENT_WIDTH} p={2} pt={4}>
+          <Box maxWidth={CONTENT_WIDTH} p={2} pt={4}>
```

Comment:
`maxWidth` is better then fixed widths for responsiveness, this resolves the layout issue on mWeb.

Reply:
This works, but it adds padding around the scroll view, which then adds some spacing around the scroll bar on the right...